src/client/src/ui/blotter/components/toolbar/QuickFilter.tsx

```diff
@@ -11,34 +11,80 @@ interface QuickFilterState {
   quickFilterText: string
 }
 
+const FilterIcon = () => (
```

Comment:
could use a filter icon from font-awesome and then use styled so it can adapt to the theme?